Author: Apollon Oikonomopoulos <apoikos@debian.org>
Description: Fix promotion to primary on startup
 Upstream commit f064a3d4 removed a call to drbdadm sh-b-pri from the
 initscript in favor of DRBD 9's auto-promotion feature. Since this feature is
 not available in the 8.4.x kernel modules, we partly revert the said commit to
 have resources configured with
 .
  startup {
    become-primary-on <node_name>;
  }
 .
 handled by the initscript.

Last-Update: 2014-07-01
Forwarded: no
--- a/scripts/drbd
+++ b/scripts/drbd
@@ -165,6 +165,7 @@
 	[ -d /var/lock/subsys ] && touch /var/lock/subsys/drbd	# for RedHat
 	$DRBDADM wait-con-int # User interruptible version of wait-connect all
 
+	$DRBDADM sh-b-pri all # Become primary if configured
 	log_end_msg 0
 	;;
     stop)
